source/sass/buyers-guide/components/header.scss

```diff
@@ -1,16 +1,13 @@
-.moz-logo {
```

Comment:
cleaned up this file a bit by moving relevant code to `primary-nav/primary-nav.scss`